docs/reverse_proxy.md

```diff
@@ -42,23 +42,53 @@ the reverse proxy and the homeserver.
             location /_matrix {
                 proxy_pass http://localhost:8008;
                 proxy_set_header X-Forwarded-For $remote_addr;
+            }            
+            location /.well-known/matrix/server {
```

Comment:
synapse doesn't serve anything here afaik (and there wouldn't be much point in it doing so).

Reply:
Maybe it can be an optional location, if I remember well Riot.im was firing errors in the console toward that location...